model/facade/action/GetPersonalSummaryByLoginDateAction.php

```diff
@@ -198,7 +208,7 @@ protected function doExecute() {
                 $weeksInBetween = $this->getWeeksInBetweenDates( $this->currentUserGoal->getInitDate(), $this->currentUserGoal->getEndDate() );
                 $extraGoalHoursSet += ( $this->currentUserGoal->getExtraHours() / $weeksInBetween );
             }
-            $originalHoursToBeWorked = round(($this->getWorkableHoursInThisJourneyPeriod() - $this->getWorkedHoursInThisJourneyPeriod())/ $this->getWeeksTillEndOfJourneyPeriod() , 2);
+            $originalHoursToBeWorked = round(($this->getWorkableHoursInThisPeriod() - $this->getWorkedHoursInThisPeriod())/ $this->getWeeksTillEndOfJourneyPeriod() , 2);
             $totalResults['weekly_goal'] = floor( ( $originalHoursToBeWorked + $extraGoalHoursSet ) * 60);
```

Comment:
You could also have done this to get the proper result:
```
$extraGoalHoursSet = (extra hours) / weeksTillEndOfJourneyPeriod
```
I still prefer the other way, because it looks simpler.